### PR TITLE
'checked' attribute should be set when string boolean like "true" is used

### PR DESCRIPTION
**Pull request by kromit**
I am using a key-value map as `intialValues`. The map is coming from server but the server has no knowledge about values so it sends all values as string. 

String `"true"` is a valid value for the `checked` attribute in react so it should also be for redux-form.

### CODE REVIEW

**Comment by erikras** [comment]
Sure, seems reasonable.

File: src/readField.js [hunk 1]
```diff
@@ -115,7 +115,7 @@ const readField = (state, fieldName, pathToHere = '', fields, syncErrors, asyncV
     const initialFormValue = read(`${name}.initial`, form);
     const initialValue = initialFormValue || read(name, initialValues);
     field.name = name;
-    field.checked = initialValue === true || undefined;
+    field.checked = String(initialValue).toLowerCase() === 'true' || undefined;
```
Comment (ooflorent) [inline src/readField.js]: I don't like this condition... A more descriptive test would have been better in my opinion, something like:

```js
initialValue === true || initialValue === "true" || undefined
```

**Comment by erikras** [comment]
Yeah, the test has failed, too. I'm repairing.

**Comment by erikras** [comment]
Published in [`v5.3.0`](https://github.com/erikras/redux-form/releases/tag/v5.3.0).

**Pull request merged**